templates/project/Cargo.toml

```diff
@@ -18,23 +18,11 @@ opt-level = "z"
 embedded-svc = "0.27"
 embedded-hal = { version = "0.2.7", features = ["unproven"]}
 log = "0.4"
-async-channel = "1.8.0"
-futures-lite = "1.12.0"
-micro-rdk = {version = "0.1.8", git = "https://github.com/viamrobotics/micro-rdk.git", features = ["esp32", "binstart"], rev = "c669b06" }
+async-channel = "2"
+futures-lite = "1"
+micro-rdk = {version = "0.1.8", git = "https://github.com/viamrobotics/micro-rdk.git", features = ["esp32", "binstart","provisioning"], rev = "e5ba39f" }
```

Comment:
The module template also has a `rev` of `c669b06` - I suspect it should get the same update.

Reply:
yes!